### PR TITLE
Move event and delayed inbox messages read to the bridge

### PR DESCRIPTION
**Pull request by yahgwai**
This makes state and events more consistent when sequencer inboxes are replaced.
* Events are emitted from the bridge
* Event signature changed due to addition of seq inbox address
* Requires a call to postUpgradeInit to initialise total delayed messages read

### CODE REVIEW

File: src/bridge/ICommon.sol [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro-contracts/blob/main/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+// solhint-disable-next-line compiler-version
+pragma solidity >=0.6.9 <0.9.0;
+
+interface ICommon {
```
Comment (gzeoneth) [inline src/bridge/ICommon.sol:8]: the name is a bit confusing, why don't we just have these in IBridge?

File: src/bridge/AbsBridge.sol [hunk 2]
```diff
@@ -55,8 +59,25 @@ abstract contract AbsBridge is Initializable, DelegateCallAware, IBridge {
...
+        uint256 indexed batchSequenceNumber,
+        bytes32 indexed beforeAcc,
+        bytes32 indexed afterAcc,
+        bytes32 delayedAcc,
+        uint256 afterDelayedMessagesRead,
+        ICommon.TimeBounds timeBounds,
+        ICommon.BatchDataLocation dataLocation
+    );
+
+    function postUpgradeInit() external onlyDelegated onlyProxyOwner {
```
Comment (gzeoneth) [inline src/bridge/AbsBridge.sol:76]: let's mention this in PR description

File: src/bridge/AbsBridge.sol [hunk 3]
```diff
@@ -55,8 +59,25 @@ abstract contract AbsBridge is Initializable, DelegateCallAware, IBridge {
 
     uint256 public override sequencerReportedSubMessageCount;
 
+    uint256 public totalDelayedMessagesRead;
+
     address internal constant EMPTY_ACTIVEOUTBOX = address(type(uint160).max);
 
+    event SequencerBatchDelivered(
```
Comment (gzeoneth) [inline src/bridge/AbsBridge.sol:66]: we should document that the signature of this event will change due to the new packing of `TimeBounds`

**Review by gzeoneth — APPROVED** [review]
LGTM, some comments

File: src/bridge/ICommon.sol (hunk 1 above)
Comment (yahgwai) [reply src/bridge/ICommon.sol:8]: removed and put in IBridge

File: src/bridge/AbsBridge.sol (hunk 3 above)
Comment (yahgwai) [reply src/bridge/AbsBridge.sol:66]: Added desc on PR

**Review by gzeoneth — APPROVED** [review]
LGTM, should remove those dot file before merging

**Pull request merged**